client/src/components/payment/payment-mixin.ts

```diff
@@ -95,12 +89,15 @@ export default class PaymentMixin extends Vue {
         'priority': priorityRequest
       })
       await paymentModule.setPaymentFees(response)
+      return true
     } catch (error) {
+      console.error('fetchFees() =', error) // eslint-disable-line no-console
       if (error instanceof PaymentApiError) {
         await errorModule.setAppError({ id: 'payment-api-error', error: error.message } as ErrorI)
       } else {
         await errorModule.setAppError({ id: 'fetch-fees-error', error: error.message } as ErrorI)
       }
+      return false
```

Comment:
I changed a lot of these methods to return success/fail (true/false), although not every calling function looks at that.

One key thing I learned is that, in a module (eg, with `@Action` decorators), you must never throw an error (that you don't catch).

Reply:
(This file is a mixin, not a module, but... consistency.)